test/gurka/test_route_incidents.cc

```diff
@@ -29,10 +29,10 @@ class IncidentsTest : public ::testing::Test {
 
     // connect the ways via the nodes
     const gurka::ways ways = {
-        {"AB", {{"highway", "tertiary"}}},  {"BC", {{"highway", "tertiary"}}},
+        {"AB", {{"highway", "tertiary"}}},  {"BC", {{"highway", "service"}}},
         {"DEF", {{"highway", "primary"}}},  {"GHI", {{"highway", "primary"}}},
         {"ADG", {{"highway", "motorway"}}}, {"BE", {{"highway", "secondary"}}},
-        {"EH", {{"highway", "service"}}},
+        {"EH", {{"highway", "tertiary"}}},
```

Comment:
With new behavior route tries to avoid service roads, so it caused changes for the cases that included `EH` edge. If there is no specific need to have the `EH` edge as a service I suggest placing that `service` class on some dead-end edge so that it does not affect routing, but still has incidents placed on it in some test cases. The alternative would be changing costings (to nullify service penalties) in all the test cases affected